src/index.scss

```diff
@@ -24,22 +24,36 @@ body {
     padding: 10px;
```

Comment:
- [ ] Beautiful work done in styling your project, kindly align your design to be the same as the one provided see screenshots below.

### Sample
![2](https://user-images.githubusercontent.com/48371156/201495190-9bf44903-6578-4720-9405-1076498b2384.png)

### Your Design
![1](https://user-images.githubusercontent.com/48371156/201495134-eafd93fc-fcb8-4800-b97c-bc0aa2da1470.png)